design-toolkit/components/src/components/tree/index.tsx

```diff
@@ -261,6 +272,15 @@ export const TreeItemContext = createContext<TreeItemContextValue>({
   ancestors: [],
 });
 
+/**
+ * TreeItem - Single node in a Tree
+ *
+ * Represents a single node within a Tree. Supports nested content, selection,
+ * expansion and optional drag handles when enabled.
+ *
+ * @example
+ * <Tree.Item id={"node-1"}>Label</Tree.Item>
```

Comment:
This component shouldn't be exported(?) and therefore shouldn't need a docblock(?)

Will have to double check to make sure that it's not being imported somewhere